securesignals-ima-devapp/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="purple_200">#FFBB86FC</color>
```

Comment:
I'm not sure these are required?

Reply:
It's needed to power the Theme which is needed to power the AppCompatActivity.

Reply:
Done.